crates/noirc_evaluator/src/ssa_refactor/acir_gen/mod.rs

```diff
@@ -676,6 +679,7 @@ impl Context {
 
                 Self::convert_vars_to_values(out_vars, dfg, result_ids)
             }
+            _ => todo!("expected a black box function"),
```

Comment:
Which functions are missing here now?

Reply:
All Intrinsic's are not fulfilled in this match case such as ArrayLen which is handled during call simplification